### ISSUE TITLE
feedback

### ISSUE THREAD

**Issue by arnaldojr** [classarnaldojr/22-4si-nac3-the-gabriels#2]
<html>
<body>
<google-sheets-html-origin><style type="text/css"></style>

  |   | R1: Exploração de Dados (Até 1 ponto) | R2: Preparação de Dados(Até 2 ponto) | R3: Desenvolvimento do Modelo (Até 1 ponto) | R4: Treinamento e Teste do Modelo (Até 2 ponto) | R5: Validação e Otimização do Modelo (Até 3 ponto) | R6: Conclusões Finais (Até 1 ponto) | R7: Deploy (Até 2 pontos *extra) | Total
-- | -- | -- | -- | -- | -- | -- | -- | -- | --
Gabriel Borowiec Marciano | Gabriel Juncken Costa | 1 | 2 | algoritimo 2 ... de progressão linearescolha errada.0,75 | 2 | 1Algoritmo 1 - gs_knn.best_params_ retorna os melhores valores {'knn__n_neighbors': 5} - esse valor não foi utlizado para retreinar o knn, foi utlizado k=7.Algoritmo 2 - escolha erradaAlgoritmo 3 - otimização não realizada , loss = 'mse' não é adequado para problemas de classificação, o certo seria 'acurracy' | 0 |   | 6,75

</body>
</html>
